### DISCUSSION TITLE
헬로스크립트

### DISCUSSION THREAD

**Discussion by almondsky** [naverz/zepeto-studio-kor#818] [5. Q&A]
안녕하세요, 
유니티/코딩 초보이다 보니 어느덧 물음표 살인마가 되어가고 있네요... 예쁘게 봐주세요 ^^ 
다름이 아니라 .. hello zepeto script 단계에 들어와 설명서와 같이 GameObject에 ZEPETOScript 추가하기를 하고 edit script 를 열어보니
<img width="1082" alt="Screen Shot 2022-05-30 at 12 12 01 AM" src="https://user-images.githubusercontent.com/105544091/170876672-cfaef3c9-b57b-4e92-bb6a-afca18f47101.png">
 설명서와 같이 이렇게 뜨지 않고 이렇게 떠서
<img width="2376" alt="Screen Shot 2022-05-30 at 12 08 30 AM" src="https://user-images.githubusercontent.com/105544091/170876686-da8447ef-cc93-4e21-a321-6f8468265ded.png">
우선 start 에 console.log(`log : Hello World`);
        console.warn(`warn : Hello World`);
        console.error(`error : Hello World`); 
집어 넣어 테스트를 해보니 출력화면에 아무것도 뜨지 않아서, 위에 스크린샷에 보이는 문구들을 아예 다 지우고 제페토 
<img width="1036" alt="Screen Shot 2022-05-30 at 12 14 00 AM" src="https://user-images.githubusercontent.com/105544091/170876779-6b8310b2-8e28-4838-aa72-e7f6e59c2c15.png">
이걸 통으로 붙여넣기를 하고 테스트를 해봐도 출력화면에 아무것도 뜨지 않는데,, 어디서 부터 잘못 된 건가요,,?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 크리에이터님 반갑습니다~
혹시 스크립트를 'Open Compiled Script'로 열어보셨을까요?
이 방법으로 오픈하게 되면 스크린샷에 올려주신 것과 같은 컴파일된 자바스크립트 형태가 뜹니다.
![스크린샷 2022-05-30 오전 10 40 44](https://user-images.githubusercontent.com/100118650/170902205-61a3aec3-dc0b-40b4-8903-1d4a55f8be44.png)


따라서 아래 방법으로 프로젝트 창에서 스크립트 파일 우클릭 > Open C# Project 또는 더블클릭으로 실행시킨 후 편집해 보세요. 
![스크린샷 2022-05-30 오전 10 42 56](https://user-images.githubusercontent.com/100118650/170902360-845cfc37-77ea-4e12-9e51-8f269a54a8bb.png)

감사합니다.

**Reply by almondsky** [reply]
해결했습니다. 감사합니다!
